### DISCUSSION TITLE
貌似日志功能有BUG？

### DISCUSSION THREAD

**Discussion by jiajiaxd** [kkkgo/PaoPaoGateWay#126] [问和答] [answered]
如图，流量记录功能一直正常的：
![image](https://github.com/user-attachments/assets/39087632-d509-4396-a17d-48b4c0e13e58)
但是日志每次刚进入一会儿，是无效的（即使我打开新的国外网站）：
![image](https://github.com/user-attachments/assets/fde4c95d-3081-4877-8489-f1b8648df017)
我如果在这个界面多停留一会儿（具体时间不确定），日志功能会恢复正常，并且开始显示记录。
我用的是paopaodns+paopaogateway分流
求教，这是BUG还是我配置的问题？

**Comment by kkkgo** [comment, marked as answer]
这个是clash核心自己的API，这里显示的是实时日志，是浏览器自己不停请求的API，不停留当然不会有记录。当然也看日志级别，如果你在设置里面点级别为debug应该刷的比较快些。
流量记录功能是ppgw自己实现的。
